features/dd-sdk-android-trace-otel/src/main/kotlin/com/datadog/android/trace/opentelemetry/OtelTracerProvider.kt

```diff
@@ -226,7 +226,8 @@ class OtelTracerProvider internal constructor(
         }
 
         /**
-         * Sets the sample rate of spans.
+         * Sets the sample rate of spans. Please have in mind that there is an automatic limit of 100
```

Comment:
so is it a change in the behavior compared to the previous tracing code version. If yes, shouldn't we disable this behavior by default to align with what we current have?

Reply:
actually I need to re - do this...we have a new config option (DD_TRACE_RATE_LIMIT) that is modifying that behaviour. I need to investigate on that.

Reply:
In case we disable it what will happen when we adopt the new `artifact` from them ? Can we still disable it ? Maybe we should keep it and align with that, I need to still see what is this `DD_TRACE_RATE_LIMIT` which is different than the `DD_TRACE_SAMPLE_RATE`